sparkles/core.py

```diff
@@ -179,9 +201,30 @@ def _run_aca_review(load_name=None, *, acars=None, make_html=True, report_dir=No
         aca.set_stars_and_mask()  # Not stored in pickle, need manual restoration
         aca.check_catalog()
 
+        # Find roll options if requested
         if roll_level == 'all' or aca.messages >= roll_level:
+            # Get roll selection algorithms to try
+            max_roll_options = roll_args.pop('max_roll_options', 10)
```

Comment:
`max_roll_options` is not mentioned in the documentation comment for `roll_args`, is this intentional?

Reply:
Fixed.

Reply:
Fixed, see 70ce3f4.